src/luac-cli.js

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/env node
+"use strict";
+
+global.WEB = false;
+
+const lua      = require('../node_modules/fengari/src/lua.js');
```

Comment:
Yes it works.
I also added the two binaries as `fengari` and `fengaric` in `package.json`.

You can test it with: `npm i -g fengari-lua/fengari-node-cli#luac`